Src/Support/Google.Apis.Core/Http/HttpClientFactory.cs

```diff
@@ -101,5 +101,29 @@ private HttpClientHandler CreateSimpleClientHandler()
             }
             return handler;
         }
+
+        /// <summary>
+        /// Create a <see cref="HttpClientHandler"/> for use when communicating with the server.
```

Comment:
I'd change "a" to "an" here as I'd sound out HttpClientHandler as aitch-tee-tee-pee client handler. YMMV :)

Reply:
done :)